src/test/scala/medeia/generic/GenericEncoderSpec.scala

```diff
@@ -31,4 +42,25 @@ class GenericEncoderSpec extends MedeiaSpec {
       ))
   }
 
+  //prevents unused field warnings
+  object ForSealedTraitWithTransformationTest {
+    sealed trait Trait
+    case class A(string: String) extends Trait
+    case class B(int: Int) extends Trait
+    implicit val coproductDerivationOptions: CoproductDerivationOptions[Trait] =
+      CoproductDerivationOptions(typeNameTransformation = { case a => a.toLowerCase() }, typeNameKey = "otherType")
```

Comment:
Can't we reuse the `Trait` and only define the implicit val for the `CoproductDerivationOptions` in the specific test?

Reply:
yes, the options have to stay in the objects nonetheless to keep the warnings down. Will do so